### ISSUE TITLE
Convert system to luatex

### ISSUE THREAD

**Issue by bhuvy2** [illinois-cs241/coursebook#80]
**Is your feature request related to a problem? Please describe.**
Right now we use pdflatex to generate docs we should use luatex

**Describe the solution you'd like**
* Change the compiler to luatex
* Remove order.tex in favor of on the fly lua generated code
* Remove all absolute imports for relative imports
* Change the math to be lua math where it both shows the equation and calculates the answer with lua -- cuts down on bugs.

**Describe alternatives you've considered**
There is MikTex and ConTex, I'd be open to hearing other alternatives than pdflatex

**Additional context**
N/A

**Comment by aneeshdurg** [comment]
as far as calculating math on the fly, I've used pgfplots for that before and it worked nicely.
The following code defines a wrapper around pgfmathprintnumber:
```
\newcommand{\calculate}[2][3]{
  \pgfkeys{/pgf/fpu}
  \pgfmathparse{#2}\pgfmathprintnumber[precision=#1]{\pgfmathresult}
  \pgfkeys{/pgf/fpu=false}
}
\newcommand{\calculatef}[2][3]{
  \pgfkeys{/pgf/fpu}
  \pgfmathparse{#2}\pgfmathprintnumber[fixed,precision=#1]{\pgfmathresult}
  \pgfkeys{/pgf/fpu=false}
}
```

I would use it like so:
```
\def\ansB{((2*(\ABlen+\BClen)/\speedOfLight)
          + \packetSize/\ABspeed + \packetSize/\BCspeed)} % in s
$\ansB = \calculate{\ansB}$s
```

luatex is more flexible so we should use that longterm, but this is a portable alternative that we could use in the interim.

**Comment by bhuvy2** [comment]
After a fair amount of searching, pandoc is at odds with evaluating luatex in a safe manner and to avoid introducing weird compatibility layers, I'll close this issue.

It would be useful if there is support down the line for a `pandoc-luatex` or similar project to bring this issue back from the grave.
